<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>limeddsconfig</class>
 <widget class="QWidget" name="limeddsconfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DDS Configuration</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>418</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Version: DDS Core v1.0</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Output Enable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="tx_mux_en_chkbx">
          <property name="text">
           <string>TxMux</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="rx_mux_en_chkbx">
          <property name="text">
           <string>RxMux</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Noise Shaper</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="taylor_en_chkbx">
          <property name="text">
           <string>Taylor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="phase_dith_chkbx">
          <property name="text">
           <string>Phase Dith</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="trunc_dith_chkbx">
          <property name="text">
           <string>Trunc Dith</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Sweep Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sweep_off_radio">
          <property name="text">
           <string>Off</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sweep_up_radio">
          <property name="text">
           <string>&amp;Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sweep_updown_radio">
          <property name="text">
           <string>Up/Down</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Sweep Rate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="sweep_rate_spin">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>2000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>MHz/ms</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="set_rate_but">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>SET</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>GET</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Freq Bot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="freq_bot_spin">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>MHz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="set_freq_bot_but">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>SET</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>GET</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Freq Top</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="freq_top_spin">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>MHz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="set_freq_top_but">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>SET</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>GET</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
